packages/react-native/React/Fabric/Mounting/ComponentViews/Modal/RCTModalHostViewComponentView.mm

```diff
@@ -218,7 +218,7 @@ - (void)boundsDidChange:(CGRect)newBounds
 
   if (_state != nullptr) {
     auto newState = ModalHostViewState{RCTSizeFromCGSize(newBounds.size)};
-    _state->updateState(std::move(newState));
+    _state->updateState(std::move(newState), EventPriority::Sync);
```

Comment:
I don't think his API is available. Were you able to compile this on the main brach?

Reply:
It should be available in ConcreteState.h. Uses `EventPriority::AsynchronousBatched)` by default